src/index.ts

```diff
@@ -900,16 +991,27 @@ export abstract class PinejsClientCore<PinejsClient> {
 	): Promise<number>;
 	public async get(
 		params: Params & { id: NonNullable<Params['id']> },
+		retryParameters?: RetryParameters,
 	): Promise<AnyObject | undefined>;
-	public async get(params: Omit<Params, 'id'>): Promise<AnyObject[]>;
-	public async get(params: Params): Promise<PromiseResultTypes> {
+	public async get(
+		params: Omit<Params, 'id'>,
+		retryParameters?: RetryParameters,
+	): Promise<AnyObject[]>;
+	public async get(
+		params: Params,
+		retryParameters?: RetryParameters,
```

Comment:
Can you change the signature so it stays as
```suggestion
		params: Params,
```
but `Params` gets `retry: RetryParameters` added please. I much prefer one object for params where we can use named params rather than the situation of multiple object parameters where it's potentially unclear which is which. (And it'll also greatly reduce the amount of changes needed for this PR)